src/app/utils/utils.ts

```diff
@@ -16,6 +16,26 @@ export const createQueryString = (params = {}) => {
     return '';
   }
 
-  return keys.map(param => `${param}=${params[param]}`).join('&');
+  return keys.map((param) => `${param}=${params[param]}`).join('&');
 };
 
+export const displayMillisecInMinSec = (ms: number) => {
+  const d = new Date(1000 * Math.round(ms / 1000));
+  let seconds = d.getUTCSeconds().toString();
+  seconds = seconds.length < 2 ? `0${seconds}` : seconds;
+  return `${d.getUTCMinutes()}:${seconds}`;
+};
+
+export const searchParamsAsObj = (str: string) => {
+  // @ts-ignore
+  return Array.from(new URL(str).searchParams).reduce((acc, [key, value]) => {
+    acc[key] = value;
+    return acc;
+  }, {});
+};
+
+export const getArtists = (artists: any[]) => {
+  return artists.length > 2
```

Comment:
unnecessary checking - you can simplify it to:
`artists.map((artist) => artist.name).join(', ')`
If you will have only 1 artist you will just get his name without comma